python/nano/tutorial/training/pytorch-lightning/lightning_cv_data_pipeline.py

```diff
@@ -15,26 +15,20 @@
 
 import torch
 import torchvision
-from turbojpeg import TurboJPEG, TJPF_GRAY
+from turbojpeg import TurboJPEG
```

Comment:
shall we add built-in support for JPEP decode support in nano? @MeouSker77

Reply:
We already have `ImageFolder`, which has built-in support. But this Oxford Pet dataset has some gif pictures, so it cannot be used with `ImageFolder`. We may need to implement our Oxford Pet dataset, and warp these codes into our Oxford Pet dataset, then this example can use our Oxford Pet dataset directly.